app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/service/VpnEnabledNotificationContentPluginPoint.kt

```diff
@@ -28,5 +28,5 @@ import com.duckduckgo.di.scopes.VpnScope
 interface VpnEnabledNotificationContentPluginPoint
 
 fun PluginPoint<VpnEnabledNotificationContentPlugin>.getHighestPriorityPlugin(): VpnEnabledNotificationContentPlugin {
-    return getPlugins().maxBy { it.getPriority().ordinal }
+    return getPlugins().filter { it.isActive() }.maxBy { it.getPriority().ordinal }
```

Comment:
Why do we need to check isActive()? Seems like all usages happens when the feature has been registered already?

Is it because onVpnStopped() could take a while to be called?

Reply:
Let's say the following scenario:
* we have another feature (in addition to AppTP) that uses the VPN service, let's call it feature "paco"
* "paco"'s priority is higher than AppTP priority
* "paco" feature is currently disabled and AppTP enabled
* without `isActive` we'll always show "paco"'s (empty) notification

does it make sense now?